Client/Frontend/Autofill/CreditCard/CreditCardSettingsView/CreditCardTableViewController.swift

```diff
@@ -161,6 +161,12 @@ extension CreditCardTableViewController: UITableViewDelegate,
 
     func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
         didSelectCardAtIndex?(viewModel.creditCards[indexPath.row])
+        guard let hostingCell = tableView.dequeueReusableCell(
+            withIdentifier: HostingTableViewCell<CreditCardItemRow>.cellIdentifier) as? HostingTableViewCell<CreditCardItemRow>
+        else {
+            return
+        }
+        hostingCell.backgroundColor = themeManager.currentTheme.colors.layer5Hover
```

Comment:
Same, so if you handle the theme inside `HostingTableViewCell` you won't have to do this in the two VCs

Reply:
Ok so turns out we don't want that 😆 Talked with Nish. There's no great solution to this